deploy/gk-deploy

```diff
@@ -233,6 +233,7 @@ while [[ $# -ge 1 ]]; do
         ;;
         -abort)
         ABORT=1
+        SKIP_PREREQ=1
```

Comment:
@ramkrsna to make it simple, we can just add the check in as mentioned by @MohamedAshiqrh . We will revisit this change later if at all required.